uat/README.md

```diff
@@ -169,14 +169,15 @@ sudo -E java -Dggc.archive=greengrass-nucleus-latest.zip -Dtest.log.path=logs -D
 Due to scenario usually requires upload/download artifacts to S3, create and delete roles, policies, do Greengrass discovery and so one please ensure CodeBuild instance has enough AWS permissions to do that.
 For more information please read [Create a CodeBuild service role](https://docs.aws.amazon.com/codebuild/latest/userguide/setting-up.html#setting-up-service-role)
 
-## Limitations
-MQTT clients based on IoT Device SDK for Java v2, mosquitto C, Paho Java, Paho Python do no provide API to get information from PUBREC/PUBREL/PUBCOMP packages used when messages published with QoS 2.
-
-Not all features of MQTT v5.0 have been implemented in clients and are supported by gRPC proto and the control as was requested, these are not bugs but designed by requirement.
-
-### Generate html-documentation from JavaDoc
+## Generate html-documentation from JavaDoc
 To generate html-documentation from JavaDoc, run the following command from the uat directory of the project:
 ```bash
+mvn -ntp -U clean install
 mvn javadoc:javadoc
 ```
-The main html-file will be located in each module by path **~/target/site/apidocs/index.html**
\ No newline at end of file
+The main html-file will be located in each module by path **target/site/apidocs/index.html**
+
+## Limitations
```

Comment:
should be "##" or "###"?

Reply:
What do you mean ?

Reply:
Limitations should be header2 because we have no header1 in that doc